### DISCUSSION TITLE
pyhss docker process not coming up

### DISCUSSION THREAD

**Discussion by ganeshg87** [herlesupreeth/docker_open5gs#215] [General]
Hi @herlesupreeth ,

   I am facing the below issue while deploying 4G VoLTE setup, 
![Screenshot from 2023-08-22 16-22-37](https://github.com/herlesupreeth/docker_open5gs/assets/83060027/244c5041-2f16-4d73-9dd5-7f30184f6664)

  It would be great to help to fix the above issue.

Thank you
Ganesh

**Comment by herlesupreeth** [comment]
I have now pushed a new commit to the repo, which compiles pyHSS from a fixed commit which I have it working locally. Please pull the latest commits and re-compile pyHSS and give it a try

**Reply by ganeshg87** [reply]
@herlesupreeth, thank you for your quick response.
 I will update and check

**Comment by ganeshg87** [comment]
@herlesupreeth , same issue was observed after the update repo also.
![Screenshot from 2023-08-22 19-58-29](https://github.com/herlesupreeth/docker_open5gs/assets/83060027/ba0c3d66-bd43-4997-9fdb-4b4d0f55d0bf)

![Screenshot from 2023-08-22 16-22-37](https://github.com/herlesupreeth/docker_open5gs/assets/83060027/4f60dde1-723e-4f42-87f5-6fd7541b5d07)

**Reply by herlesupreeth** [reply]
Then, I would suggest the following

1. Bring down all the components
2. Then execute, `docker system prune -f`
3. Then, remove the volume as follows: `docker volume rm docker_open5gs_dbdata`
4. Then, bring up all the components

**Reply by ganeshg87** [reply]
thanks a lot.

 4G deployment successfully. I am trying to validate the E2E IMS call.
